spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1538,7 +1538,9 @@ For Spring Cloud Stream samples, please refer to the https://github.com/spring-c
 
 == Getting Started
 
-To get started with creating Spring Cloud Stream applications, visit the https://start.spring.io[Spring Initializr] and create a new Maven project named "GreetingSource".  Note that Spring Cloud Stream applications are known to work with Spring Boot versions from 1.3.x to {supported-spring-boot-version}. In the _Search for dependencies_ text box type `Stream Rabbit` or `Stream Kafka` depending on what binder you want to use.
+To get started with creating Spring Cloud Stream applications, visit the https://start.spring.io[Spring Initializr] and create a new Maven project named "GreetingSource".
+Select Spring Boot {supported-spring-boot-version} in the dropdown.
```

Comment:
It is from [here] (https://github.com/spring-cloud/spring-cloud-stream/pull/677/files#diff-c289cb8e00827701d86a1312504841a3R106)